app.js

```diff
@@ -1,17 +1,83 @@
 /* Imports */
-// import { getRandomItem } from './utils.js';
+import { getRandomItem } from './utils.js';
 
 /* State */
+let gameState = 'guess'; // 'guess' or 'results'
+let guess = ''; // 'left' 'middle' or 'right'
+let pearl = ''; // 'true' or 'false'
+let result = ''; //'win' or 'lose'
 
 /* Actions */
-function loadPage() {}
+function loadPage() {
+    displayShells();
+    // displayResults();
+}
+
+const choices = ['left', 'middle', 'right'];
+
+function findPearl(userGuess) {
+    gameState = 'results';
+    guess = userGuess;
+    pearl = getRandomItem(choices);
+}
```

Comment:
good start, you still need to check if ```guess === pearl``` and use that to update ```result``` and use result to decide what the browser will render